src/mainwindow.cpp

```diff
@@ -625,7 +660,11 @@ void MainWindow::on_actionOpen_beta_log_txt_triggered()
 
 void MainWindow::on_actionMerge_into_tab_triggered()
 {
-    QStringList files = QFileDialog::getOpenFileNames(this, "Select one or more log files to merge into current tab", "", "Log Files (*.txt *.log);;All Files (*)");
+    QFileDialog fileDlg(this, "Select one or more log files to merge into current tab", GetOpenDefaultFolder(), "Log Files (*.txt *.log);;All Files (*)");
+    fileDlg.setFileMode(QFileDialog::ExistingFiles);
+    fileDlg.exec();
+    QStringList files = fileDlg.selectedFiles();
+    m_lastOpenFolder = fileDlg.directory().absolutePath();
     TreeModel * model = GetCurrentTreeModel();
```

Comment:
Maybe consider a helper function to use in both places.

Reply:
put it into a fubnction called PickLogFilesToOpen